### DISCUSSION TITLE
The relation collection doesn't exist

### DISCUSSION THREAD

**Discussion by TheHypnoo** [pocketbase/pocketbase#6055] [Q&A] [answered]
I am making the relevant changes to the migrations to make it work with 0.23.x instead of 0.22.x . I am doing it manually, the thing is that I have a file for each collection, but in one of the schema I have a relation field that the collectionId does not exist at that time. 

What should I do to prioritise the migration of one collection from another, or would the solution be to just do it all in one file? Before with 0.22.x this didn't happen to me. 

If necessary I will pass you the code of both collections

Thank you.

**Comment by ganigeorgiev** [comment, marked as answer]
> What should I do to prioritise the migration of one collection from another, or would the solution be to just do it all in one file? Before with 0.22.x this didn't happen to me.

Change the order of the code that inserts the collections. With v0.22.x this didn't happen because there was no validations. v0.23.x applies validation by default (if you don't want validations, change `app.Save` with `app.SaveNoValidate` but I don't recommend doing it since you may end up with invalid/broken state if not used with proper care and understanding).

I recommend following the upgrade steps mentioned in the overview of:
- Go https://pocketbase.io/v023upgrade/go/#upgrade-overview
- JSVM https://pocketbase.io/v023upgrade/jsvm/#upgrade-overview
